src/components/single-line-diagram.js

```diff
@@ -98,7 +98,7 @@ const SingleLineDiagram = (props) => {
                 .addTo(divElt)
                 .size(svgWidth, svgHeight)
                 .viewbox(xOrigin, yOrigin, svgWidth, svgHeight)
-                .panZoom({panning: true, zoomMin: 0.5, zoomMax: 10, zoomFactor: 0.2, margins: {top: svgHeight/4, left: svgWidth/4, bottom: svgHeight/4, right: svgWidth/4}});
```

Comment:
On my mouse, to have a "normal" experience, I need to use zoomFactor = 10 or 20 !  What does it do on your mouse ??

Reply:
even 200 feels natural

Reply:
With my mouse, when i set the zoom factor to 10, it zooms a lot with only one wheel step ...

Reply:
Maybe we need to add a parameter next to the "center" and "diagonal labels" ?

Reply:
why not ...